OP-1043 | Update users and user groups doc (#260)
informatici/openhospital-doc: doc_user/UserManual.adoc

image:image195.png[User Browser,width=621,height=305]

The column, *LOCKED*, if checked indicates that the user is currently locked out from logging into the system.

> The last column, *DISABLED*, if checked indicates that the user has been marked as deleted(in that case, it cannot login into the application).

Reviewer: ```suggestion
The last column, *DELETED*, if checked indicates that the user has been marked as deleted and cannot be assigned to new users.
```

Author: How can an user `be assigned to new users` ?

Reviewer: Sorry, my bad, I thought we were talking about UserGroupos. Anyway the term you also used is "Deleted" (not "Disabled") and the phrase looks like too similar to "Locked" definition. What about this?

```suggestion
The last column, *DELETED*, if checked indicates that the user has been marked as deleted and, unlike the locked user, it cannot even try to login.
```

The functions available in this window are:

* *Select group*: filter the list by selecting only one group.
* *[.underline]##N##ew*: enter a new user (cannot use the same name). Valid characters are a-z, 0-9, dash, period and underscore.
* *[.underline]##E##dit*: edit an existing user (the password cannot be changed).
* *[.underline]##R##eset Password*: change the password for an existing user.
* *[.underline]##D##elete*: delete an existing user. If the user is already used in the system, then the application will prompt you to mark it as deleted instead.
* *[.underline]##C##lose*: close the window.

By pressing the *[.underline]##E##dit* button, the *_Edit User_* window is shown:

image:image198.png[Edit User]

In editing mode, the group, the description, whether the user is marked as deleted and whether the user is locked out from logging into the system can be modified.
To *lock* or *unlock* a user account check or uncheck the box after *Locked:*.

> To mark an user as deleted or restore it from deleted status, check or uncheck the box after *Deleted:*.

Reviewer: ```suggestion
To mark a user as deleted or restore it from deleted status, check or uncheck the box after *Deleted:*.
```
